### ISSUE TITLE
Hemingway Rewritten theme - Main Accent color

### ISSUE THREAD

**Issue by masperber** [Automattic/wp-calypso#66762]
### Quick summary

In the past, the Main Accent color chosen from the customizer in the Hemingway Rewritten theme would affect the background under the Site Title and Tagline text and under the menu.

Currently, those colors are set to `#1d1d1d` regardless of the value set for the color in the customizer.

### Steps to reproduce

1. Start a site with the Hemingway Rewritten theme
2. Go to the customizer and change the Main Accent color

### What you expected to happen

The Main Accent color should affect the background color for the Site Title and Tagline and for the menu.

### What actually happened

The Main Accent color only affects link and button colors, from what I can tell.

### Context

Customer report: 36538737-hc

### Platform (Simple, Atomic, or both?)

Simple, Atomic

### Theme-specific issue?

Hemingway Rewritten

### Reproducibility

Consistent

### Severity

All

### Available workarounds?

No but the platform is still usable

**Comment by masperber** [comment]
- [ ] 5491205-zen Customer requests follow-up when the bug is fixed.

**Comment by Robertght** [comment]
📌 HOUSEKEEPING
- Labels ✅ 
- Priority ✅ 
- Replicable on Core - No

📌 SCRUBBING
- Tested on P2 - no
- Tested on Simple ✅ 
- Tested on AT ✅ 
- Tested on Self-hosted - no

📌 FINDINGS/SCREENSHOTS/VIDEO
- I'm able to replicate the described behavior, but I do not see any proof that this was changed and I do not see any changes made on the theme in the past few months. 
- https://web.archive.org/ doesn't work right now therefore, I cannot compare a previous version of the site.

📌 ACTIONS
- Await author feedback

📌 Message to Author
 - @masperber, when https://web.archive.org/ works again, could you have a look and share a screenshot of how it used to look like please?

**Comment by kerrynicl** [comment]
Here's how the site looked before:
<img width="1668" alt="Markup 2022-08-25 at 07 52 30" src="https://user-images.githubusercontent.com/83325600/186669866-5a1e9a1f-06a5-4e31-bb65-7ffad715e481.png">
vs now
<img width="1874" alt="Markup 2022-08-25 at 07 53 34" src="https://user-images.githubusercontent.com/83325600/186670091-d1d03b11-1478-4f3e-980c-2c59457720b4.png">

**Comment by Robertght** [comment]
Thank you @kerrynicl !

@masperber, @kerrynicl - I checked and this is because of a missing CSS code from the user's site. Here's the missing code:

```
.site-branding{background:#006400;opacity:.8;display:inline-block;padding:.5em}.main-navigation{background:#006400}.main-navigation li:before{color:#fff}.main-navigation ul ul li{background:#fff}.main-navigation a,.main-navigation a:visited{color:#fff}.main-navigation li:hover > a{color:#999}.main-navigation .current_page_item a,.main-navigation .current-menu-item a{color:#fff}#header h1{padding-top:100px}

````

I identified this by checking the styling source through the https://web.archive.org/ site. Here's the file:

https://web.archive.org/web/20220105192530cs_/https://s1.wp.com/?custom-css=1&csblog=56ish&cscache=6&csrev=19

I'm closing this for now, but please inform the user about the above.

**Comment by liviopv** [comment]
@Robertght Uhm, it might not be a bug; there's certainly something weird about this.

The link you mentioned above has a date of 2022-01-05, but the site's Custom Design upgrade expired in November of 2015, and there's no trace that the site had permission to load custom CSS during this period. 

Could it be a glitch where Custom Design was removed, but the site kept loading the CSS regardless?

**Comment by Robertght** [comment]
Thank you @liviopv ! That is indeed strange and I took some time to check some things. I added the Custom CSS upgrade back to it and the CSS code is back as well on it. 

While our logs show the upgrade being removed, it most likely wasn't removed from the site itself as the blog report audit doesn't show a log of the Custom Design being removed from the site. 

I believe this might've been triggered by another fix from here though: https://github.com/Automattic/wp-calypso/issues/65498

Closing it for now.
